src/app/organisations/views/organisations.ejs

```diff
@@ -151,30 +151,30 @@
                                     <% if (organisation.approvers && organisation.approvers.length === 0) { %>
                                         No approvers
                                     <% } else if (organisation.approvers) { %>
-                                        <a class="govuk-link-bold govuk-link--no-visited-state toggle-open" href="">
-                                            Show approvers
-                                            <span class="govuk-visually-hidden">(for <%= organisation.name %>)</span>
-                                        </a>
-                                        <div class="meta govuk-visually-hidden">
-                                            <div class="approvers">
-                                                <dl class="inline condensed small-dt">
+                                        <div class="dfe-custom-details-component">
+                                            <details class="govuk-details govuk-!-font-size-16">
+                                                <summary class="govuk-details__summary">
+                                                    <span class="govuk-details__summary-text" data-open="Hide approvers" data-close="Show approvers">
+                                                    </span>
+                                                </summary>
+                                                <div class="govuk-details__text">
                                                     <ul>
                                                         <% for (let a = 0; a < organisation.approvers.length; a++) { %>
-                                                            <li><%= organisation.approvers[a].email %></li>
+                                                            <li>
+                                                                <%= organisation.approvers[a].email %>
+                                                            </li>
                                                         <% } %>
-                                                    </ul>
-                                                </dl>
-                                            </div>
+                                                    </ul>                                              
```

Comment:
Can you remove the random extra spaces in this line?

Reply:
@insacuri the white spaces were removed